### PR TITLE
fritzing: 0.9.4-498 -> 0.9.6

### PR DESCRIPTION
**Pull request by musfay**
###### Motivation for this change
A new version of Fritzing is released.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

This version of fritzing is properly tagged (not CD-XXX). Removed a merged patch and updated parts to latest commit.

### CODE REVIEW

**Comment by musfay** [comment]
I added `cc-by-sa-30` as most of the parts are licensed under CC-BY-SA.

File: pkgs/applications/science/electronics/fritzing/default.nix [hunk 1]
```diff
@@ -5,41 +5,36 @@
...
+  version = "0.9.6";
 
   src = fetchFromGitHub {
-    owner = "fritzing";
-    repo = "fritzing-app";
-    rev = "CD-${fritzingBuild}";
-    sha256 = "0aljj2wbmm1vd64nhj6lh9qy856pd5avlgydsznya2vylyz20p34";
+    owner  = "fritzing";
+    repo   = "fritzing-app";
+    rev    = "${version}";
```
Comment (SuperSandro2000) [inline pkgs/applications/science/electronics/fritzing/default.nix:23]:
```suggestion
    rev    = version;
```

Comment (SuperSandro2000) [inline pkgs/applications/science/electronics/fritzing/default.nix:23]: Please revert this as nixpkgs-fmt would revert it anyway.

File: pkgs/applications/science/electronics/fritzing/default.nix [hunk 2]
```diff
@@ -5,41 +5,36 @@
...
   parts = fetchFromGitHub {
-    owner = "fritzing";
-    repo = "fritzing-parts";
-    name = "fritzing-parts";
-    rev = partsSha;
-    sha256 = "0spka33a5qq34aq79j01arw1aly4vh0hzv7mahryhdlcdk22qqvc";
+    owner  = "fritzing";
+    repo   = "fritzing-parts";
+    name   = "fritzing-parts";
+    rev    = "${partsSha}";
```
Comment (SuperSandro2000) [inline pkgs/applications/science/electronics/fritzing/default.nix:31]:
```suggestion
    rev    = partsSha;
```

File: pkgs/applications/science/electronics/fritzing/default.nix [hunk 3]
```diff
@@ -5,41 +5,36 @@
...
+    sha256 = "1f4w0hz44n4iw1rc5vhcgzvlji54rf4yr8bvzkqv99hn2xf5pjgs";
   };
 
   buildInputs = [ qtbase qtsvg qtserialport boost libgit2 ];
-
   nativeBuildInputs = [ qmake pkg-config qttools ];
 
-  patches = [(fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/applications/science/electronics/fritzing/default.nix:37]: Please remove the unused fetchpatch from inputs.

**Comment by musfay** [comment]
Removed `fetchpatch` and ran nixpkgs-fmt

**Review by SuperSandro2000 — APPROVED** [review]
diff LGTM

**Comment by musfay** [comment]
We don't need to use script to build Fritzing. Generated Makefile gives better output than script. We don't need to install parts manually.

**Pull request merged**